Source/Game/Discord/DiscordBotManager.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+using Discord.WebSocket;
+
+namespace Game.Discord
+{
+    public static class DiscordBotManager
```

Comment:
Seems like a pointless class to me? Or is this meant to be used in the future? :stuck_out_tongue:

Reply:
Maybe in the future can be used, i kept this just to remember where is the place to handle received messages!